views/agency-requests/agency_requests_results.php

```diff
@@ -36,28 +37,35 @@
     <ul class="list-group" style="max-width: 700px; width: 100%; margin: 0 auto">
         <?php
         #if query returns nothing, throw an error to the user
-        if(pg_num_rows($result) == 0){
+         if(pg_num_rows($result) == 0){
         ?>
-            <div class="alert alert-warning">
-                <strong>Sorry, did not return any results for </strong> <u><?php echo $searchquery ?></u>
-            </div>
+			<div class="w-100 d-flex flex-column justify-content-center text-center">
+				<h3 class="display-3 text-secondary" style="font-size: 40px;">
+					<i class="fa fa-exclamation-circle"></i>
+				</h3>
+				<h3 class="display-3 text-secondary" style="font-size: 40px;">No Participants Found.</h3>
+			</div>
         <?php
         }//end if
+		
         #check the result of query and loop through each row to display search results
         while ($row = pg_fetch_assoc($result)) {
+		
             ?>
+
             <li class="list-group-item">
-                <button class="btn btn-outline-info advanced-info"><i class="fa fa-question" aria-hidden="true"></i></button>
-                <span>&nbsp;<?php echo $row['lastname'].", ".$row['firstname']. " ". $row['middleinit'];?></span>
-                <a class="float-right" href="/view-participant/<?= $row['participantid'] ?>">
+                <button class="btn btn-outline-secondary advanced-info"><i class="fa fa-caret-right" aria-hidden="true"></i></button>
+
+                <span><?php echo $row['lastname'].", ".$row['firstname']. " ". $row['middleinit'];?></span>
```

Comment:
You can change this to `<?= $row['lastname'] . ", " . $row['firstname'] . " " . $row['middleinit'] ?>`